pydantic/validators.py

```diff
@@ -511,6 +525,12 @@ def find_validators(  # noqa: C901 (ignore complexity)
         )
 
 
+def _get_arbitrary_class(type_):
+    origin = getattr(type_, '__origin__', None)
+    if origin is not None and issubclass(origin, Type):  # type: ignore
+        return type_.__args__[0]  # type: ignore
```

Comment:
we should also support use of bare `Type`, eg.

```py
class Foo(BaseModel):
    bar: Type
```

Reply:
What would that mean?  that is has to be a class but could be any class?

Reply:
yes, as specified by python typing.

I think `isinstance(v, type)` is the simplest check.

Reply:
quick question:

```
>>>Type
typing.Type
>>>Type.__args__
(+CT_co,)
```
it seems that the args of Type (without []) is `+CT_co`) 
have you seen this before? it does not seem to be very google friendly

Reply:
no I haven't, but it looks like you can do

```py
typing.Type.__args__[0] == typing.CT_co
```

Or to work with 3.6

```py
typing.Type.__args__ is None or typing.Type.__args__[0] == typing.CT_co
```

Reply:
see https://github.com/python/cpython/blob/b9f932f9e2a170a8d39b3c17f5fabb0967839d85/Lib/typing.py#L1185

Reply:
it says CT_co not meant for export, so I tried to get along without it (also mypy complains). 
Have a look now!